bip-0000.mediawiki

```diff
@@ -0,0 +1,289 @@
+<pre>
+  BIP: ?
+  Layer: Applications
+  Title: Ordinal Numbers
+  Author: Casey Rodarmor <casey@rodarmor.com>
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/casey/ord/discussions/126
+  Status: Draft
+  Type: Informational
+  Created: 2022-02-02
+  License: CC0-1.0
+</pre>
+
+== Introduction ==
+
+=== Abstract ===
+
+This document defines a scheme for assigning serial numbers to sats.
+
+=== Copyright ===
+
+This work is placed in the public domain.
+
+=== Motivation ===
+
+Bitcoin has no notion of stable, public accounts or identities. Addresses are
+single-use, and wallet accounts are private. Additionally, the use of addresses
+or public keys as stable identifiers precludes transfer of ownership or key
+rotation.
+
+This proposal is motivated by the desire to provide stable identifiers that may
+be used by Bitcoin applications.
```

Comment:
> wallet accounts are private

"wallet accounts"? Is that a thing?

> "the use of addresses or public keys as stable identifiers precludes transfer or ownership or key rotation"

What does this mean?

> desire to provide stable identifiers that may be used by Bitcoin applications

Why not justify the need from within the system instead of from outside? Maybe something like <<unequivocally identifying the original system's currency units ("sats") is necessary for easily determining ownership in an eventual smaller currency units system ("sub-sats")>> replacing this paragraph?

Reply:
> "wallet accounts"? Is that a thing?

You can suggest a better thing to rephrase it